### ISSUE TITLE
Upgrading from 1.0.1 to 1.0.3 breaks homebridge on my side

### ISSUE THREAD

**Issue by julien94800** [homebridge/homebridge#2540]
Hello,

I tried to update homebridge from 1.0.1 to 1.0.3 this morning and now no way to launch it again. I always get the following message. Any idea on where I should investigate? Even in Debug mode, no way to have some more information. I tried to remove the persist folder without any success

Thanks in advance

[**Logs:**]()
Error: EISDIR: illegal operation on a directory, read
    at Object.readSync (fs.js:524:3)
    at tryReadSync (fs.js:349:20)
    at Object.readFileSync (fs.js:386:19)
    at LocalStorage.parseDirSync (/usr/local/lib/node_modules/homebridge/node_modules/node-persist/src/local-storage.js:637:35)
    at LocalStorage.parseDataDirSync (/usr/local/lib/node_modules/homebridge/node_modules/node-persist/src/local-storage.js:572:21)
    at LocalStorage.initSync (/usr/local/lib/node_modules/homebridge/node_modules/node-persist/src/local-storage.js:145:14)
    at new Server (/usr/local/lib/node_modules/homebridge/src/server.ts:116:22)
    at cli (/usr/local/lib/node_modules/homebridge/src/cli.ts:52:18)
    at Object.<anonymous> (/usr/local/lib/node_modules/homebridge/bin/homebridge:17:22)
    at Module._compile (internal/modules/cjs/loader.js:1156:30)

**Environment:**

* **Node.js Version**: 12.16.2
* **NPM Version**: 6.14.4
* **Homebridge Version**: 1.0.3
* **Operating System**: Docker on Synology
* **Process Supervisor**: Docker on Synology

**Comment by nicoduj** [comment]
if you are on synology, check there is no @eadir directories hidden in your mounted directory for homebridge coker. Had the same issue and took a while to figure it out ...

**Comment by julien94800** [comment]
Hello @nicoduj 
Thanks for the hint. I only saw your message but I succeed to fix before this even if not sure how. I noticed that some files from persist and accessories folder were from "user" permission whereas in some older folders it was from "root" permission. I restored the permission and now it seems to work. I really do not understand what happened. I now tried to reload several times and it is still working.....
